metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -449,3 +449,15 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: InvalidStorageAutoScalerCRsFound
```

Comment:
@aruniiird how about `StorageAutoScalerCRIsInvalid`?

Reply:
Ok, made the change...
PS: was trying to avoid `Is` / `Was` and other verbs (=